postgres/dev.sls

```diff
@@ -3,12 +3,22 @@
 {% if grains.os not in ('Windows', 'MacOS',) %}
 
   {% if postgres.pkg_dev %}
+
+    {% if grains.os_family == 'RedHat' and postgres.version|int >= 10 %}
```

Comment:
Is that really applicable for PG 10 only? I haven't looked for it myself, so just wondering.